### ISSUE TITLE
Potentailly Wrong Number Of Props In Musical5

### ISSUE THREAD

**Issue by foohyfooh** [kwsch/PKHeX#4063]
**Describe the bug**

There maybe incorrect information documented for the number of props in the code and thus the final flags assignment may be wrong. On [Bulbapedia Prop Page](https://bulbapedia.bulbagarden.net/wiki/Prop) it lists 100 props and the only place that I saw which has 101 props is [Serebii Musical Page](https://www.serebii.net/blackwhite/pokemonmusical.shtml) but that is a mistake since it lists 2 items twice and is missing the top hat.
https://github.com/kwsch/PKHeX/blob/aec0367085957c6a9e48caa4ea38e3835e9fec45/PKHeX.Core/Saves/Substructures/Gen5/Musical5.cs#L10-L17

This means that the last line of the unlock all props method might needs to be changed to 
```csharp
Data[bitFieldOffset + 0xC] = 0x0F; // top 4 bits unset, to complete multiple of 8 (100=>104 bits).
```

**Comment by kwsch** [comment]
Notes:
Text dump has lines 3-102 filled, only 100 unique entries.
https://projectpokemon.org/rawdb/black2/msg/93.php

**Comment by sora10pls** [comment]
Closed by #4069
